### ISSUE TITLE
lib/libltdl.so.7.3.0 is bundled here

### ISSUE THREAD

**Issue by isuruf** [conda-forge/gct-feedstock#4]
This library is from libtool.

**Comment by chrisburr** [comment]
Thanks for noticing,  I'll deal with it soon.

**Comment by chrisburr** [comment]
Fix in #3:
```
   INFO (gct,bin/globus-url-copy): Needed DSO lib/libltdl.so.7 found in conda-forge::libtool-2.4.6-h516909a_1005
```
Thanks again!
